gravity-forms/gw-cache-buster.php

```diff
@@ -177,8 +177,13 @@ public function shortcode( $markup, $attributes, $content ) {
 					// GF is using it as their standard for triggering the `gform_post_render` event, I figured we should follow suit.
 					gform.initializeOnLoaded( function() {
 						// Form has been rendered. Trigger post render to initialize scripts.
-						jQuery( document ).trigger( 'gform_post_render', [ formId, 1 ] );
-						gform.utils.trigger({ event: 'gform/postRender', native: false, data: { formId: formId, currentPage: 1 } });
+						<?php
+							echo 'gform.initializeOnLoaded( function() {' . GFFormDisplay::post_render_script(
+								$form_id,
+								GFFormDisplay::get_current_page( $form_id )
+								) .
+								'} );';
+						?>
```

Comment:
```suggestion
						<?php
							$post_render_script = GFFormDisplay::post_render_script( $form_id, GFFormDisplay::get_current_page( $form_id ) );
							print_f( 'gform.initializeOnLoaded( function() { %s } )', $post_render_script );
						?>
```

Dude, please don't hate me for making a formatting suggestion but I find `print_f` so much easier to read, particularly when there's a multi-line function in play. Thoughts on something like this?

Reply:
Haha no sweat 😉  Change made